docs/application/dotnet/guides/nui/view.md

```diff
@@ -10,11 +10,13 @@
 `Position`, `Orientation`, and `Scale` of `View` are known as transforms.
 `View` supports hierarchical structure. Therefore, `Position`, `Orientation`, and `Scale` of every `View` are affected by their parents.
 
-`Position` of each `View` is defined as the distance between the position of [ParentOrigin](https://samsung.github.io/TizenFX/latest/api/Tizen.NUI.BaseComponents.View.html#Tizen_NUI_BaseComponents_View_ParentOrigin) and **TopLeft** position of `View`. `ParentOrigin` is a property that defines the reference point in the parent `View`. For example, if a `View` object has properties `Position` (5, 3) and `ParentOrigin` is **Center**, then the **TopLeft** of the object is located (5, 3) away from **Center** position of its parent `View` object.
+`Position` of each `View` is defined as the distance between the position of [ParentOrigin](https://samsung.github.io/TizenFX/latest/api/Tizen.NUI.BaseComponents.View.html#Tizen_NUI_BaseComponents_View_ParentOrigin) and [PivotPoint](https://samsung.github.io/TizenFX/latest/api/Tizen.NUI.BaseComponents.View.html#Tizen_NUI_BaseComponents_View_PivotPoint) of the `View`. The `ParentOrigin` is a property that defines the reference point in the parent `View`, whereas the `PivotPoint` is a propetry that defines the anchor point of the child, the default value is **TopLeft**. For example, if a `View`'s property `Position` is set to (5, 3), `ParentOrigin` property is set to **Center** and `PivotPoint` property is set to **TopLeft**, then the **TopLeft** of the object is located (5, 3) coordinates away from the **Center** position of its parent `View` object.
 
-In addition, if the property `ParentOrigin` is changed to **BottomRight**, then the position of `View` is moved to (5, 3) from the **BottomRight** position of its parent. NUI provides predefined `ParentOrigin` as shown in the following figure:
+In addition, if the property `ParentOrigin` property is changed to **BottomRight**, then the position of the **TopLeft** corner of the `View` will be located (5, 3) coordinates from the **BottomRight** corner. 
 
-**Figure: The position of predefined ParentOrigin**
+NUI provides predefined `ParentOrigin` as well as `PivotPoint` as shown in the following figure. You can also use custom values.
```

Comment:
```suggestion 
```suggestion
NUI provides predefined `ParentOrigin` as well as `PivotPoint` positions as shown in the following figure, and you can also use custom values for setting the position:
```

I have done a slight modification, request you to kindly check for technical accuracy.

Reply:
I have incorporated your suggestions and fixed mistakes.